proxy/envoy/policy.go

```diff
@@ -61,6 +61,7 @@ func insertMixerFilter(listeners []*Listener, instances []*model.ServiceInstance
 							"source.ip":  context.IPAddress,
 							"source.uid": context.UID,
 						},
+						QuotaName: "RequestCount",
```

Comment:
I feel like I would vote for moving back to `Check`-based quota enforcement until we have this configuration story figured out. It seems really really odd to go through adding the quota descriptor bits into a scope/subject rule and then not have that work because I need to do the same thing somewhere else (even within the context of alpha). The value of having in that scope/subject config seems completely lost.

For that matter, I'm still confused as to why `RequestCount` is set to 5 QPS globally (and is not configured to 5 QPS inside a selector config), but perhaps that is another matter.